### PR TITLE
chore: add latest dogfood template

### PR DESCRIPTION
**Pull request by bpmct**
https://github.com/coder/coder/issues/6452

### CODE REVIEW

**Review by matifali — APPROVED** [review]

File: dogfood/main.tf [hunk 1]
```diff
@@ -157,14 +236,14 @@ resource "docker_container" "workspace" {
   image = docker_image.dogfood.name
   name  = local.container_name
   # Hostname makes the shell more user friendly: coder@my-workspace:~$
-  hostname   = data.coder_workspace.me.name
-  entrypoint = ["sh", "-c", coder_agent.dev.init_script]
+  hostname = data.coder_workspace.me.name
+  # Use the docker gateway if the access URL is 127.0.0.1
+  entrypoint = ["sh", "-c", replace(coder_agent.dev.init_script, "/localhost|127\\.0\\.0\\.1/", "host.docker.internal")]
```
Comment (matifali) [inline dogfood/main.tf:241]: you may remove the unnecessary replace

**Review by matifali — APPROVED** [review]

**Review by matifali — APPROVED** [review]

**Comment by matifali** [comment]
I would send a sperate PR to update RAM and, hopefully CPU usage.

**Review by kylecarbs — APPROVED** [review]

**Comment by matifali** [comment]
Seems like `CODER_SESSION_TOKEN` is expired too.

**Comment by matifali** [comment]
@bpmct would you like to merge this as #7424 has been resolved by @ammario

**Pull request merged**